lua/entities/gmod_wire_light.lua

```diff
@@ -136,7 +139,7 @@ function ENT:Initialize()
 end
 
 function ENT:Directional( On )
-	if On then
+	if On and self:GetOn() then
```

Comment:
There shouldn't be a GetOn check here, this is a function for creating the entity.